### DISCUSSION TITLE
Bagaimana Cara Membatasi Tag yang Muncul di Laman?

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#179] [Q&A]
```
<?php foreach($page->tags as $tag): ?>
<?=$tag->link?>
<?=$tag->title?>
<?php endforeach; ?>
```
Kalau saya tulis seperti diatas, hasilnya, semua tag yang terdapat pada halaman akan muncul.
Misal:
`label1 label2 label3 label4`
Bagaimana cara membatasi tag yang muncul, misalkan kita hanya ingin menampilkan 1 atau 2 tag, atau kita hanya ingin memunculkan tag tertentu
Misal: (saya ingin membatasi jumlah yang muncul)
`label1` => saya hanya ingin menampilkan **label1**

**Comment by taufik-nurrohman** [comment]
> Bagaimana cara membatasi tag yang muncul, misalkan kita hanya ingin menampilkan 1 atau 2 tag&hellip;

Coba begini:

~~~ php
<?php $limit = 5; ?>
<?php foreach ($page->tags->chunk($limit, 0) as $tag): ?>
  <a href="<?= $tag->link; ?>" rel="tag">
    <?= $tag->title; ?>
  </a>
<?php endforeach; ?>
~~~

> atau kita hanya ingin memunculkan tag tertentu&hellip;

Coba begini:

~~~ php
<?php $name = 'slug-tag'; ?>
<?php if ($tag = $page->tags[$name]): ?>
  <a href="<?= $tag->link; ?>" rel="tag">
    <?= $tag->title; ?>
  </a>
<?php endif; ?>
~~~

**Reply by yuliakbard** [reply]
```
<?php $name = 'slug-tag'; ?>
<?php if ($tag = $page->tags[$name]): ?>
  <a href="<?= $tag->link; ?>" rel="tag">
    <?= $tag->title; ?>
  </a>
<?php endif; ?>
```
yang ini gak bisa kang

**Reply by taufik-nurrohman** [reply]
Nilai `slug-tag` sudah diganti?

**Reply by yuliakbard** [reply]
udah kang, tetap gak mau, rusak tampilan websitenya jadinya

**Reply by taufik-nurrohman** [reply]
Yang ini tak jadiin _bug_ aja.
